sample/src/main/kotlin/com/slack/circuit/sample/home/HomeNavPresenter.kt

```diff
@@ -41,7 +40,7 @@ object HomeNavScreen : Screen {
 @Composable
 fun homeNavPresenter(events: Flow<HomeNavScreen.Event.HomeNavEvent>): HomeNavScreen.HomeNavState {
   var state by remember {
-    mutableStateOf(HomeNavScreen.HomeNavState(bottomNavItems = homeScreenNavItems))
+    mutableStateOf(HomeNavScreen.HomeNavState(index = 0, bottomNavItems = HOME_NAV_ITEMS))
```

Comment:
Is there a benefit of setting the index here as opposed to when writing the data class?

Reply:
index 0 seems like a reasonable default, as opposed to being whatever index the adoptable item is (and having the adoptable item _know_ its own index)